07-asymmauth.tex

```diff
@@ -191,8 +191,8 @@ \subsection{Schlüssel- und Signaturerzeugung}
 Zahl $e \in \{1, \dots, p - 1\}$ gewählt,
 wobei $p=|\G|$. Damit berechnet man 
 \begin{align*}
-a &:= g^e \in \mathbb{G}\\
-b &:= (M - a \cdot x) \cdot e^{-1} \mod p
+a &:= g^e \mod p \in \mathbb{G}\\
```

Comment:
Ich war einfach verwirrt durch die ganzen Definitionen. Also p ist hier mit p := |G| ja schon p-1 (in meinem Kopf war die Gruppe in der wir sind Z_p mit p prim).